client/src/layout/navbar-layout/NavbarLayout.tsx

```diff
@@ -30,28 +40,88 @@ const NavbarLayout: React.FC<INavbarLayout> = ({ collapse, setCollapse }) => {
                     className={styles.iconCollapse}
                     onClick={() => setCollapse(!collapse)}
                 />
-                <Typography.Title className={styles.logo}>
+                <Link to="./" className={styles.logo}>
                     9GAP
-                </Typography.Title>
+                </Link>
             </div>
             <div className={styles.itemContainer}>
-                <Button
-                    shape="circle"
-                    icon={<SearchOutlined />}
-                    className={styles.iconCollapse}
-                />
+                <Popover
+                    placement="bottom"
+                    trigger="click"
+                    content={
+                        <div className={styles.searchContainer}>
+                            <Input
+                                className={styles.searchInput}
+                                bordered={false}
+                                size="large"
+                                placeholder="Search..."
+                                onPressEnter={(e) => console.log(e.currentTarget.value)}
+                                prefix={
+                                    <SearchOutlined />
+                                }
+                            />
+                        </div>
+                    }
+                >
+                    <Button
+                        shape="circle"
+                        icon={<SearchOutlined />}
+                        className={styles.iconCollapse}
+                    />
+                </Popover>
+                <Popover
+                    placement="bottom"
+                    trigger="click"
+                    content={
+                        <div className={styles.notifyContainer}>
+                        </div>
+                    }
+                >
+                    <Button
+                        shape="circle"
+                        icon={<BellFilled />}
+                        className={styles.iconCollapse}
+                    />
+                </Popover>
                 <Typography.Text
                     className={styles.text}
                     onClick={protectAction(() => console.log("Authenticated"))}
```

Comment:
Please remember to remove all the `console.log` in code before opening a code review.